README.md

```diff
@@ -14,7 +14,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live De Link](https://marcraphael12.github.io/Marc-Raphael-Portofolio/)
+[Take a nice look of this projec here](https://marcraphael12.github.io/Marc-Raphael-Portofolio/)
```

Comment:
- [ ]  The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.

- Add a descriptive summary,
- Add instructions on how to clone and run the project locally,
- Since you're the only author on the project, there's no need to add a `author1` replace it with just `author`,
- Fix the typo you have on the highlighted line above.